docs/quick_start/starting.rst

```diff
@@ -15,3 +15,16 @@ The following command options are available:
 For example, to start the Fledge system, open a session to the Fledge device and type::
 
 /usr/local/fledge/bin/fledge start
+
+If authentication is enabled, which is the default mode for Fledge version 3.0 onward, then the commands can be passed a username, using the -u flag and will prompt for a password for that user.
```

Comment:
can be executed with a username using the -u flag, and you will be prompted to enter the password for that user.